gradle-jdks/build.gradle

```diff
@@ -44,21 +46,26 @@ gradlePlugin {
             displayName = 'Palantir JDK Auto-provisioning Plugin'
             description = 'Auto-provisions specific versions of JDKs'
             implementationClass = 'com.palantir.gradle.jdks.JdksPlugin'
+            tags.addAll("java", "jdks")
+        }
+        jdksIdea {
+            id = 'com.palantir.jdks-idea'
```

Comment:
Probably call this `com.palantir.jdks.idea`, using dots is the style we already use (like `com.palantir.jdks.latest` or `com.palantir.jdks.internal.xxxx`).